### PR TITLE
Fix traefik configuration. Add ClassicUI route.

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/devops/stacks/plone.yml [hunk 1]
```diff
@@ -97,15 +97,30 @@ services:
         - traefik.enable=true
         - traefik.docker.network=traefik-public
         - traefik.constraint-label=traefik-public
+        # AUTHENTICATION
+        # Note: all dollar signs in the hash need to be doubled for escaping.
+        # To create user:password pair, it's possible to use this command:
+        # echo $(htpasswd -nb user password) | sed -e s/\\$/\\$\\$/g
+        # Defaults to admin:admin
+        - traefik.http.middlewares.backend-auth.basicauth.users=admin:$$apr1$$uZPT5Fgu$$AmlIdamxT5ipBvPlsdfD70
```
Comment (jensens) [inline {{ cookiecutter.project_slug }}/devops/stacks/plone.yml:105]: I would use an environment variable for the pw ash instead of hard-coding it in here.

**Comment by sneridagh** [comment]
@jensens @ericof no clue why the 3.8 tests are failing.

**Comment by jensens** [comment]
> @jensens @ericof no clue why the 3.8 tests are failing.

yo was not found in the pre checks, maybe related?

File: {{ cookiecutter.project_slug }}/devops/stacks/plone.yml (hunk 1 above)
Comment (sneridagh) [reply {{ cookiecutter.project_slug }}/devops/stacks/plone.yml:105]: Makes sense, but I won't have time for it :( I'm fixing things in a hurry whilst I'm working on the training docs.

Should I remove the ClassicUI and the basic passwd thing in the meanwhile? The other fix is quite important... The rewrite is not done well without it.

**Comment by sneridagh** [comment]
And in 3.9 and 3.10 they are ok? Quite strange.

**Comment by jensens** [comment]
Indeed. I have no idea.

**Comment by sneridagh** [comment]
@jensens it seems it was a test glitch, I rerun and now it's green.

Regarding your suggestion, can we leave it as a "nice to have"? Maybe something we can improve at the conference sprint?

**Pull request merged**